packages/react/menu/src/Menu.tsx

```diff
@@ -554,6 +569,12 @@ const MenuContentImpl = React.forwardRef((props, forwardedRef) => {
                       searchRef.current = '';
                     }
                   })}
+                  onMouseMove={composeEventHandlers(props.onMouseMove, (event) => {
+                    const target = event.target as HTMLElement;
+                    if (event.currentTarget.contains(target) && event.movementX !== 0) {
+                      pointerDirRef.current = event.movementX >= 1 ? 'right' : 'left';
```

Comment:
just a nit here, but if we're already checking for `!== 0` above, I feel like `> 0` here would express the directional intent better, rather than re-encapsulating that non-0 check.